### PR TITLE
Link 36 more Starbucks

### PR DESCRIPTION
**Pull request by validcube**
# Description
https://docs.google.com/spreadsheets/d/1AXc9EDXA6udZeGROtB5nuABjM33VluGY_V24tIzHaKc

Whoever decided it was a good idea to make a separate application for each country, I hate you

## Icons addition information

**Starbucks**: I was so so wrong thinking that they all use the same activity name...

#### Starbucks (**Starbucks**)

* com.starbucks.ch/**Starbucks**
* com.starbucks.co/**Starbucks**
* com.starbucks.in/**Starbucks**
* com.starbucks.mx/**Starbucks**
* com.starbucks.my/**Starbucks**
* com.starbucks.tr/**Starbucks**
* com.starbucks.br/**Starbucks**
* com.starbucks.id/**Starbucks**
* com.starbucks.au/**Starbucks**
* com.starbucks.cee/**Starbucks**
* com.starbucks.de/**Starbucks**
* com.starbucks.uk/**Starbucks**
* com.starbucks.jp/**Starbucks**
* com.starbucks.za/**Starbucks**
* com.starbucks.ar/**Starbucks**
* com.starbucks.mobilecard/**Starbucks**
* com.starbucks.id/**Starbucks**
* com.starbucks.cl/**Starbucks**
* com.starbucks.tw/**Starbucks**
* com.starbucks.singapore/**Starbucks**
* com.starbucks.ph/**Starbucks**

#### Not in the spreadsheet 
* com.starbucks.thailand/**Starbucks**
* com.starbucks.nz/**Starbucks**
* com.starbucks.fr/**Starbucks**
* com.starbucks.ru/**Starbucks**
* com.starbucks.ae/**Starbucks**
* com.starbucks.kw/**Starbucks**
* com.starbucks.pt/**Starbucks**
* com.starbucks.sa/**Starbucks**

    Starbucks Kazakhstan
    Starbucks Cambodia
    Starbucks Sweden
    Starbucks Ireland
    Starbucks Vietnam
    Starbucks Hong Kong
    Starbucks Peru

And more...

## Contributor's checklist
- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

**Comment by validcube** [comment]
😂 I'll let you know when one of the APK do the funny and changes the App Activity to be different from the others.

If not, I'll just mark this as ready for review

File: app/assets/appfilter.xml [hunk 1]
```diff
@@ -5543,7 +5543,37 @@
...
+  <item component="ComponentInfo{com.starbucks.ae/com.starbucks.ae.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.ru/com.starbucks.ru.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.fr/com.starbucks.fr.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.nz/com.starbucks.nz.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.thailand/com.starbucks.thailand.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.ph/com.starbucks.ph.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.singapore/com.starbucks.singapore.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
```
Comment (validcube) [inline app/assets/appfilter.xml:5556]: Starbucks Singapore

```suggestion
  <item component="ComponentInfo{com.starbucks.singapore/com.starbucks.sg.MainActivity}" drawable="starbucks" name="Starbucks" />
```

File: app/assets/appfilter.xml [hunk 2]
```diff
@@ -5543,7 +5543,37 @@
...
+  <item component="ComponentInfo{com.starbucks.pw/com.starbucks.pw.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.pt/com.starbucks.pt.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.kw/com.starbucks.kw.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.ae/com.starbucks.ae.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.ru/com.starbucks.ru.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.fr/com.starbucks.fr.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.nz/com.starbucks.nz.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
```
Comment (validcube) [inline app/assets/appfilter.xml:5553]: Starbucks New Zealand

```suggestion
  <item component="ComponentInfo{com.starbucks.nz/com.starbucks.nz.MainActivity}" drawable="starbucks" name="Starbucks" />
```

File: app/assets/appfilter.xml [hunk 3]
```diff
@@ -5543,7 +5543,37 @@
   <item component="ComponentInfo{me.tylerbwong.stack/me.tylerbwong.stack.ui.MainActivity}" drawable="stack" name="Stack" />
   <item component="ComponentInfo{com.standardnotes/com.standardnotes.MainActivity}" drawable="standard_notes" name="Standard Notes" />
   <item component="ComponentInfo{com.vitotechnology.StarWalk2Free/com.google.firebase.MessagingUnityPlayerActivity}" drawable="star_walk_2" name="Star Walk 2" />
-  <item component="ComponentInfo{com.starbucks.cn/com.starbucks.cn.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="starbucks" />
+  <item component="ComponentInfo{com.starbucks.sa/com.starbucks.sa.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.pw/com.starbucks.pw.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
```
Comment (validcube) [inline app/assets/appfilter.xml:5547]: Invalid (Missing APK)

```suggestion
```

File: app/assets/appfilter.xml [hunk 4]
```diff
@@ -5543,7 +5543,37 @@
...
+  <item component="ComponentInfo{com.starbucks.ph/com.starbucks.ph.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.singapore/com.starbucks.singapore.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.tw/com.starbucks.tw.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.cl/com.starbucks.cl.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.id/com.starbucks.id.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.mobilecard/com.starbucks.mobilecard.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.ar/com.starbucks.ar.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
```
Comment (validcube) [inline app/assets/appfilter.xml:5561]: Starbucks Argentina

```suggestion
  <item component="ComponentInfo{com.starbucks.ar/com.starbucks.emea.features.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
```

File: app/assets/appfilter.xml [hunk 5]
```diff
@@ -5543,7 +5543,37 @@
   <item component="ComponentInfo{me.tylerbwong.stack/me.tylerbwong.stack.ui.MainActivity}" drawable="stack" name="Stack" />
   <item component="ComponentInfo{com.standardnotes/com.standardnotes.MainActivity}" drawable="standard_notes" name="Standard Notes" />
   <item component="ComponentInfo{com.vitotechnology.StarWalk2Free/com.google.firebase.MessagingUnityPlayerActivity}" drawable="star_walk_2" name="Star Walk 2" />
-  <item component="ComponentInfo{com.starbucks.cn/com.starbucks.cn.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="starbucks" />
+  <item component="ComponentInfo{com.starbucks.sa/com.starbucks.sa.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
```
Comment (validcube) [inline app/assets/appfilter.xml:5546]: Invalid (APK Missing)

```suggestion
```

File: app/assets/appfilter.xml [hunk 6]
```diff
@@ -5543,7 +5543,37 @@
...
+  <item component="ComponentInfo{com.starbucks.id/com.starbucks.id.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.br/com.starbucks.br.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.tr/com.starbucks.tr.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.my/com.starbucks.my.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.mx/com.starbucks.mx.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.in/com.starbucks.in.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.co/com.starbucks.co.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
```
Comment (validcube) [inline app/assets/appfilter.xml:5574]: Starbucks South Korea

```suggestion
  <item component="ComponentInfo{com.starbucks.co/com.starbucks.co2.ui.intro.IntroActivity}" drawable="starbucks" name="Starbucks" />
```

File: app/assets/appfilter.xml [hunk 7]
```diff
@@ -5543,7 +5543,37 @@
...
+  <item component="ComponentInfo{com.starbucks.ch/com.starbucks.ch.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="Starbucks" />
```
Comment (validcube) [inline app/assets/appfilter.xml:5575]: Starbucks Switserland

```suggestion
  <item component="ComponentInfo{com.starbucks.ch/com.starbucks.emea.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
```

**Review by validcube — COMMENTED** [review]
Correct Starbucks activity FOR ALL STARBUCKS

File: app/assets/appfilter.xml [hunk 8]
```diff
@@ -5543,7 +5543,34 @@
   <item component="ComponentInfo{me.tylerbwong.stack/me.tylerbwong.stack.ui.MainActivity}" drawable="stack" name="Stack" />
   <item component="ComponentInfo{com.standardnotes/com.standardnotes.MainActivity}" drawable="standard_notes" name="Standard Notes" />
   <item component="ComponentInfo{com.vitotechnology.StarWalk2Free/com.google.firebase.MessagingUnityPlayerActivity}" drawable="star_walk_2" name="Star Walk 2" />
-  <item component="ComponentInfo{com.starbucks.cn/com.starbucks.cn.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="starbucks" />
+  <item component="ComponentInfo{com.starbucks.pt/com.starbucks.emea.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
```
Comment (validcube) [inline app/assets/appfilter.xml:5546]: * Starbucks Kazakhstan
* Starbucks Cambodia 
* Starbucks Sweden
* Starbucks Ireland
* Starbucks Vietnam

```suggestion
  <item component="ComponentInfo{com.starbucks.vn/com.starbucks.vn.MainActivity}" drawable="starbucks" name="Starbucks" />
  <item component="ComponentInfo{com.starbucks.ie/com.starbucks.emea.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
  <item component="ComponentInfo{com.starbucks.se/com.coniq.app.activity.SplashActivity}" drawable="starbucks" name="Starbucks" />
  <item component="ComponentInfo{com.starbucks.kh/com.starbucks.kh.MainActivity}" drawable="starbucks" name="Starbucks" />
  <item component="ComponentInfo{com.starbucks.kz/ru.ideast.starbucks.modules.welcome.ui.activities.LaunchActivity}" drawable="starbucks" name="Starbucks" />
  <item component="ComponentInfo{com.starbucks.pt/com.starbucks.emea.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
```

File: app/assets/appfilter.xml [hunk 9]
```diff
@@ -5543,7 +5543,39 @@
   <item component="ComponentInfo{me.tylerbwong.stack/me.tylerbwong.stack.ui.MainActivity}" drawable="stack" name="Stack" />
   <item component="ComponentInfo{com.standardnotes/com.standardnotes.MainActivity}" drawable="standard_notes" name="Standard Notes" />
   <item component="ComponentInfo{com.vitotechnology.StarWalk2Free/com.google.firebase.MessagingUnityPlayerActivity}" drawable="star_walk_2" name="Star Walk 2" />
-  <item component="ComponentInfo{com.starbucks.cn/com.starbucks.cn.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="starbucks" />
+  <item component="ComponentInfo{com.starbucks.vn/com.starbucks.vn.MainActivity}" drawable="starbucks" name="Starbucks" />
```
Comment (validcube) [inline app/assets/appfilter.xml:5546]: * Starbucks Hong Kong
* Starbucks Peru

```suggestion
  <item component="ComponentInfo{ccom.starbucks.pe/com.starbucks.emea.features.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
  <item component="ComponentInfo{com.starbucks.hk/com.starbucks.MainActivity}" drawable="starbucks" name="Starbucks" />
  <item component="ComponentInfo{com.starbucks.vn/com.starbucks.vn.MainActivity}" drawable="starbucks" name="Starbucks" />
```

Comment (x9136) [reply app/assets/appfilter.xml:5548]: Most likely you need to change `ccom` to `com`.

If you still have all these apps installed, please make sure the icon is applied to all of them.

**Review by x9136 — CHANGES_REQUESTED** [review]

File: app/assets/appfilter.xml [hunk 10]
```diff
@@ -5543,7 +5543,41 @@
...
+  <item component="ComponentInfo{com.starbucks.uk/com.starbucks.emea.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.de/com.starbucks.emea.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.cee/com.starbucks.cee.view.MainActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.au/au.com.mobileden.starbucksau.activities.SplashActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.id/com.starbucks.id.MainActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.br/com.starbucks.br.MainActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.tr/com.starbucks.tr.ui.splash.ACSplash}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.my/com.starbucks.revamp.activity.SplashActivity}}" drawable="starbucks" name="Starbucks" />
```
Comment (x9136) [inline app/assets/appfilter.xml:5575]:
```suggestion
  <item component="ComponentInfo{com.starbucks.my/com.starbucks.revamp.activity.SplashActivity}" drawable="starbucks" name="Starbucks" />
```

File: app/assets/appfilter.xml [hunk 11]
```diff
@@ -5543,7 +5543,41 @@
   <item component="ComponentInfo{me.tylerbwong.stack/me.tylerbwong.stack.ui.MainActivity}" drawable="stack" name="Stack" />
   <item component="ComponentInfo{com.standardnotes/com.standardnotes.MainActivity}" drawable="standard_notes" name="Standard Notes" />
   <item component="ComponentInfo{com.vitotechnology.StarWalk2Free/com.google.firebase.MessagingUnityPlayerActivity}" drawable="star_walk_2" name="Star Walk 2" />
-  <item component="ComponentInfo{com.starbucks.cn/com.starbucks.cn.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="starbucks" />
+  <item component="ComponentInfo{ccom.starbucks.pe/com.starbucks.emea.features.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
```
Comment (x9136) [inline app/assets/appfilter.xml:5546]:
```suggestion
  <item component="ComponentInfo{com.starbucks.pe/com.starbucks.emea.features.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
```

File: app/assets/appfilter.xml [hunk 12]
```diff
@@ -6369,7 +6369,41 @@
...
+  <item component="ComponentInfo{com.starbucks.uk/com.starbucks.emea.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.de/com.starbucks.emea.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.cee/com.starbucks.cee.view.MainActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.au/au.com.mobileden.starbucksau.activities.SplashActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.id/com.starbucks.id.MainActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.br/com.starbucks.br.MainActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.tr/com.starbucks.tr.ui.splash.ACSplash}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.my/com.starbucks.revamp.activity.SplashActivity}}" drawable="starbucks" name="Starbucks" />
```
Comment (validcube) [inline app/assets/appfilter.xml:6401]:
```suggestion
  <item component="ComponentInfo{com.starbucks.my/com.starbucks.revamp.activity.SplashActivity}" drawable="starbucks" name="Starbucks" />
```

Co-author marked

File: app/assets/appfilter.xml [hunk 13]
```diff
@@ -6369,7 +6369,41 @@
...
+  <item component="ComponentInfo{com.starbucks.thailand/com.sbth_mobile_app.MainActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.ph/com.starbucks.revamp.activity.SplashActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.singapore/com.starbucks.sg.MainActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.tw/com.starbucks.tw.activity.SplashActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.cl/com.starbucks.mx.splash.SBSplashActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.mobilecard/com.starbucks.mobilecard.main.activity.LandingPageActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.ar/com.starbucks.emea.features.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
+  <item component="ComponentInfo{com.starbucks.za/com.starbucks.emea.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
```
Comment (validcube) [inline app/assets/appfilter.xml:6392]:
```suggestion
  <item component="ComponentInfo{com.starbucks.za/com.starbucks.emea.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
  <item component="ComponentInfo{com.starbucks.sa/com.starbucks.emea.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
```

Co-author (SB KSA)

File: app/assets/appfilter.xml [hunk 14]
```diff
@@ -6369,7 +6369,41 @@
   <item component="ComponentInfo{me.tylerbwong.stack/me.tylerbwong.stack.ui.MainActivity}" drawable="stack" name="Stack" />
   <item component="ComponentInfo{com.standardnotes/com.standardnotes.MainActivity}" drawable="standard_notes" name="Standard Notes" />
   <item component="ComponentInfo{com.vitotechnology.StarWalk2Free/com.google.firebase.MessagingUnityPlayerActivity}" drawable="star_walk_2" name="Star Walk 2" />
-  <item component="ComponentInfo{com.starbucks.cn/com.starbucks.cn.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="starbucks" />
+  <item component="ComponentInfo{ccom.starbucks.pe/com.starbucks.emea.features.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
```
Comment (validcube) [inline app/assets/appfilter.xml:6372]:
```suggestion
  <item component="ComponentInfo{com.starbucks.pe/com.starbucks.emea.features.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
```

Co-author marked

**Comment by x9136** [comment]
Are you planning to make any other changes, or do you want me to finalize the PR and merge?

**Review by x9136 — APPROVED** [review]

**Comment by validcube** [comment]
> Are you planning to make any other changes, or do you want me to finalize the PR and merge?

I'm done, feel free to merge this. :+1:

**Pull request merged**